tech/languages/csharp/mono-installation.md

```diff
@@ -1,24 +1,44 @@
 ---
-title: Mono
+title: C#
 subsection: csharp
 section: tech-languages
-order: 1
+order: 4
 version: 7.8.4
-description: "C# provided by Mono: a cross-platform, open source, .NET development framework"
+description: "C# - .NET Core and Mono"
```

Comment:
In this case, I see a title, not a description.
Here are some examples from other pages:

```
An open source programming language built to craft simple, reliable, and efficient software.
-
General-purpose, object-oriented and concurrent computer programming language.
-
Server-side HTML embedded scripting language.
-
Highly capable, multi-purpose and feature-rich programming language.
```